src-docs/src/views/datagrid/datagrid.js

```diff
@@ -312,6 +312,10 @@ export default () => {
     console.log(eventData);
   });
 
+  const onFullScreenChange = useRef((eventData) => {
+    console.log('isFullScreen:', eventData);
```

Comment:
Other events already use `console.log` in this file. I think this work could be a followup

Reply:
@AMoo-Miki I agree that `console.log` is not ideal. Especially for those who don't have their console open. We could add a toast notification for every change, but the table is already transitioning between full and normal screen states. `console.log` demo is intended to simplify the review process, rather than being a user facing feature